packages/@coorpacademy-components/.storybook/preview.js

```diff
@@ -1,23 +1,57 @@
-const React = require('react');
-const {default: createTranslate} = require('@coorpacademy/translate');
-const en = require('../locales/en/global');
-const {default: Provider} = require('../src/atom/provider');
-const skin = require('./skin');
+import React from 'react';
+import createTranslate from '@coorpacademy/translate';
+import en from '../locales/en/global';
+import fr from '../locales/fr/global';
+import ja from '../locales/ja/global';
+import ko from '../locales/ko/global';
+import es from '../locales/es/global';
+import vi from '../locales/vi/global';
+import Provider from '../src/atom/provider';
 
-const translate = createTranslate(en);
+import skin from './skin';
 
-const context = {
-  skin,
-  translate,
-  Vimeo: window.Vimeo
+const withProvider = (Story, context) => {
+  const locales = {
+    en,
+    fr,
+    ja,
+    ko,
+    es,
+    vi
+  };
+
+  const providerContext = {
+    skin,
+    translate: createTranslate(locales[context.globals.locale]),
+    Vimeo: window.Vimeo,
+  };
+
+  return (
+    <Provider {...providerContext}>
+      <Story />
+    </Provider>
+  );
 };
 
-module.exports = {
-  decorators: [
-    Story => (
-      <Provider {...context}>
-        <Story />
-      </Provider>
-    )
-  ]
+export const globalTypes = {
+  locale: {
+    name: 'Locale',
+    description: 'Internationalization locale',
+    defaultValue: 'fr',
+    toolbar: {
+      icon: 'globe',
+      items: [
+        { value: 'fr', right: '🇫🇷', title: 'Français' },
+        { value: 'en', right: '🇺🇸', title: 'English' },
+        { value: 'es', right: '🇪🇸', title: 'Español' },
+        { value: 'ja', right: '🇨🇳', title: 'Japanese' },
+        { value: 'ko', right: '🇰🇷', title: '한국어' },
+        { value: 'vi', right: '🇻🇳', title: 'vietnamese' },
+      ],
```

Comment:
Let's check with @audric-coorp or @wernerdurand If any other language deserves to give some special attention regarding the UI